sailthru_content/services/sailthru_translation_service.py

```diff
@@ -58,11 +60,28 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             for seat in course_run['seats']:
                 sailthru_content_vars['price_{}'.format(seat['type'])] = seat['price']
                 sailthru_content_vars['currency_{}'.format(seat['type'])] = seat['currency']
+                # Adding the course type with the precedence as
+                # Credit, Professional, Verified, Audit so that if a course run have
+                # multiple seat types the one with the higher precedence will be added
+                # in course_type
+                if not sailthru_content_vars.get('course_type', None):
+                    sailthru_content_vars['course_type'] = seat['type']
+                if sailthru_content_vars['course_type'] == 'audit' or sailthru_content_vars['course_type'] == 'honor':
```

Comment:
Ah, I didn't notice this code is in a loop on `seats`. We can certainly make this code easier. Something like

```
seat_priority=['professional', 'verified', 'no-id-professional', 'audit', 'honor']
if seat_priority.index(sailthru_content_vars['course_type']) < seat_priority.index(seat['type']):
    sailthru_content_vars['course_type'] = seat['type']
```